__tests__/checkVersion.test.ts

```diff
@@ -14,12 +14,20 @@ describe('checkCppVersion', () => {
   });
 
   it('checks version successfully', () => {
-    expect(checkCppVersion).not.toThrow();
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    jest.spyOn(console, 'warn').mockImplementation(() => {});
```

Comment:
Is there some kind of built-in `jest.id` function so we can skip disabling linter here?